lib/cretonne/meta/isa/intel/recipes.py

```diff
@@ -1021,6 +1022,32 @@ def floatccs(iform):
         modrm_r_bits(out_reg0, bits, sink);
         ''')
 
+#
+# Conditional move (a.k.a integer select)
+# (maybe-REX.W) 0F 4x modrm(r,r)
+# 1 byte, modrm(r,r), is after the opcode
+#
+cmov = TailRecipe(
+        'cmov', IntSelect, size=1, ins=(FLAG.eflags, GPR, GPR), outs=1,
+        requires_prefix=False,
+        clobbers_flags=False,
+        emit='''
+        PUT_OP(bits | icc2opc(cond), rex2(in_reg2, in_reg1), sink);
+        modrm_rr(in_reg2, in_reg1, sink);
+        ''')
```

Comment:
This is where I think you'll need `outs=2` instead of `outs=1` so the output register is the same as the when-false input register.